### ISSUE TITLE
TypeError: Cannot read property 'body' of undefined - Fragment in *.graphql file does't load

### ISSUE THREAD

**Issue by fokot** [apollographql/graphql-tag#64]
You can take a look on example here https://github.com/fokot/GitHunt-React/commit/962e568486a3d0eebb0ddceae88b78a432ae981a

```
import PostedByFragmentFromFile from './PostedByFragment.graphql';
 
 const PostedByFragmentInline = gql`
     fragment PostedByFragment on Entry {
       postedBy {
          html_url
          login
       }
     }
   `;
 
  RepoInfo.fragments = {
    entry: gql`
      fragment RepoInfo on Entry {
          stargazers_count
          open_issues_count
        }
      ...PostedByFragment
      }
    ${PostedByFragmentFromFile}
    `,
  };
  
// ^^^ PostedByFragmentFromFile does not work but PostedByFragmentInline works
```

If fails with error
```
result += args[i].loc.source.body;
TypeError: Cannot read property 'body' of undefined
```
Also botch fragment looks almost the same.

**Comment by jnwng** [comment]
@fokot thanks for reporting — after poring through your fork, i was able to reproduce the issue, and distilled it down to this line: 
https://github.com/apollographql/graphql-tag/blob/master/loader.js#L27-L28
```
  const doc = gql`${source}`;
  const outputCode = `var doc = ${JSON.stringify(doc)};`;
```
basically (after checking with @stubailo), since [source is now no longer enumerable](https://github.com/graphql/graphql-js/pull/463#issuecomment-239994165), when we `stringify` the doc, we lose that `source` that is causing this failure. this is coming up now because in graphql-tag@1.3.0 we [bumped our dependency on graphql](https://github.com/apollographql/graphql-tag/blob/1.3.0/CHANGELOG.md#v130) from 0.6.x to 0.9.x.

since the webpack loader requires the intermediary "codegen" step, i don't think we'll be able to use `source.body` anymore... but not sure what to replace it with just yet. in the short term, instead of using 
```
import PostedByFragmentFromFile from './PostedByFragmentFromFile.graphql';
```
you may be able to just use the _inline_ version of the import, like
```
gql`
  #import "./PostedByFragment.graphql"
  fragment RepoInfo on Entry {
      stargazers_count
      open_issues_count
    ...PostedByFragment
  }
`

can you give that a try?

**Comment by jnwng** [comment]
here's a super hacky short-term fix that i drummed up: https://github.com/jnwng/graphql-tag/commit/739722d32272cfaf02726be768a54005c0ebc568 that should unblock you while we figure out a better way to get this to work properly.

**Comment by fokot** [comment]
I was in hurry, so my workaround was using just plain js files but I will definitelly move to new graphql-tag when this is resolved:
```
>>> /PostedByFragment.js
export default gql`
   fragment PostedByFragment on Entry {
     postedBy {
        html_url
        login
     }
   }
 `;
```

**Comment by jnwng** [comment]
this should be resolved by #65 and is available in v2.1.0
